etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Elgentos_ServerSideAnalytics" setup_version="0.1.0">
+    <module name="Elgentos_ServerSideAnalytics" setup_version="0.1.1">
```

Comment:
I don't think you need to update this, since you're using `db_schema.xml`.

Reply:
You're right, updated 🚀